### DISCUSSION TITLE
Add to Inpaint or Outpaint Image Prompt

### DISCUSSION THREAD

**Discussion by VlasovFM** [lllyasviel/Fooocus#1478] [Ideas]
Please, add to "Inpaint or Outpaint" the ability to add Image Prompt, which will only affect the selected area of the main image

**Comment by R4290N** [comment]
it's already done.  just use "Mixing Image Prompt and Inpaint"

**Comment by VlasovFM** [comment]
Hello. Understood. Thank you for the information. Then add the ability to
erase some area painted over with a brush in "Inpaint or Outpaint" . Like
an eraser

пн, 18 дек. 2023 г., 00:05 R4290N ***@***.***>:
